frontend/packages/cli/rollup.config.js

```diff
@@ -16,12 +16,23 @@ export default {
       preferBuiltins: true,
       extensions: ['.mjs', '.js', '.json', '.node', '.ts'],
     }),
-    typescript({
-      outputToFilesystem: true,
-      tsconfig: './tsconfig.node.json',
+    swc({
+      jsc: {
+        parser: {
+          syntax: 'typescript',
+        },
+        target: 'es2022',
+      },
```

Comment:
https://github.com/liam-hq/liam/pull/2299#discussion_r2184830677
@FunamaYukina I don't think this change has anything to do with the comment above, is it necessary? This is a separate area from Drizzle Parser and probably should not be changed.

Reply:
This setting was unnecessary. I removed it.🙏
https://github.com/liam-hq/liam/pull/2299/commits/5156a98d7007318177b9712c383a09c657b4117d